GNUmakefile

```
@@ -115,9 +115,14 @@ link-git-hooks: ## Install git hooks
 update-atlas-sdk: ## Update the atlas-sdk dependency
 	./scripts/update-sdk.sh
 
-# details on usage can be found in CONTRIBUTING.md under "Creating New Resource and Data Sources"
+# details on usage can be found in CONTRIBUTING.md under "Scaffolding initial Code and File Structure"
 .PHONY: scaffold
 scaffold:
 	@go run ./tools/scaffold/*.go $(name) $(type)
 	@echo "Reminder: configure the new $(type) in provider.go"
 
+# details on usage can be found in CONTRIBUTING.md under "Scaffolding Schema and Model Definitions"
+.PHONY: scaffold-schemas
+scaffold-schemas:
+	@tools/scaffold/scripts/schema-scaffold.sh $(name)
```

```suggestion
	@tools/scaffold/scripts/schema-scaffold.sh $(resource_name)
```

In my PR for generating doc https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1829 I am using resource_name to add additional context. let me know if you have a strong opinion in using `name` instead

---

`resource_name` sounds good. This is a small nit: both scaffold commands receive the `resource_name` in camel case, while in docs receives it is in snake case. I think the formats received in both cases are out of convenience, just stating in case it is something simple to align.

---

unfortunately, we are forced to use the snake case for tfplugindocs. We could rename the name in the script but it would require additional code to handle it